mods/d2k/rules/arrakis.yaml

```diff
@@ -9,8 +9,7 @@ spicebloom.spawnpoint:
 		GrowthTerrainTypes: SpiceSand
 		SpawnActor: spicebloom
 		GrowthSequences: grow0
-		GrowthDelay: 250, 750
-		RespawnDelay: 1, 2
+		GrowthDelay: 1750, 3250
```

Comment:
The default RespawnDelay was `1500, 2500`, together with the old GrowthDelay that totaled up to 1750 min / 3250 max as well (ignoring that 1-2 extra ticks).